dotcom-rendering/src/model/insertPromotedNewsletter.ts

```diff
@@ -72,14 +94,26 @@ const getDistanceAfterFloating = (
 				['supporting', 'thumbnail', 'richLink'].includes(element.role),
 		);
 
+	// it would be more logical to return Infinity here,
+	// but having an outsized finite value simplifies the sort function
+	// (Infinity - Infinity == NaN)
+	const outsizedValue = elements.length * 10;
```

Comment:
Same as above regarding JSDoc.

Would `elements.length + 1` be enough, as it’s simply one more than the amount of elements? Otherwise, maybe you want [`Number.MAX_VALUE`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_VALUE)

Reply:
`elements.length +1` would be enough; it just needs to be a number higher than the function could otherwise return. I'll use that instead